Harvest.Core/Services/SlothService.cs

```diff
@@ -59,8 +60,12 @@ public async Task<SlothResponseModel> MoveMoney(int invoiceId)
                 .Include(a => a.Project).ThenInclude(a => a.Accounts).SingleOrDefaultAsync();
             if (invoice == null)
             {
-                Log.Error("Invoice not found: {invoiceId}", invoiceId);
-                return null;
+                return Result.Error("Invoice not found: {invoiceId}", invoiceId);
+            }
+
+            if (invoice.Project.Accounts.Count == 0)
```

Comment:
Can't get to this currently?

Reply:
It did occur, though it was probably old data that won't happen again with current checks in place.